schemas/json/theme.json

```diff
@@ -2770,7 +2796,7 @@
 		"version": {
 			"description": "Version of theme.json to use.",
 			"type": "integer",
-			"enum": [ 3 ]
+			"const": 3
```

Comment:
Is it worth changing this to const as well?

https://github.com/WordPress/gutenberg/blob/update/json-schema-draft-07/schemas/json/theme.json#L2782

Reply:
> Is it worth changing this to const as well?
> 
> [`update`/json-schema-draft-07/schemas/json/theme.json#L2782](https://github.com/WordPress/gutenberg/blob/update/json-schema-draft-07/schemas/json/theme.json?rgh-link-date=2024-07-20T10%3A41%3A03Z#L2782)

Yeah I'll update it. I wonder why the automate migration didn't update it?

Reply:
> Yeah I'll update it. I wonder why the automate migration didn't update it?

The automation didn't catch it because it was something I added after the automation. And I don't want to update it because the intention is for this `enum` to catch the remaining `propertyNames` that aren't part of the variable. In this case there's only one, but making it `const` would be inconsistent with every other instance of the `propertyNames` that we have.